pkg/module/template.go

```diff
@@ -41,7 +41,7 @@ spec:
 `
 )
 
-func Template(remote ocm.ComponentVersionAccess, moduleTemplateName, channel string, data []byte, labels, annotations map[string]string) ([]byte, error) {
+func Template(remote ocm.ComponentVersionAccess, moduleTemplateName, namespace string, channel string, data []byte, labels, annotations map[string]string) ([]byte, error) {
```

Comment:
Can we start to create some unit tests for this func? Should be nice and easy to make it table driven?